rocket/runModelStepTest.py

```diff
@@ -55,6 +58,12 @@
 
 m.solve(Remote=False)
```

Comment:
"remote=False" should be in the model initialization in the latest gekko version.
m=GEKKO(remote = False)